junifer/markers/utils.py

```diff
@@ -44,3 +49,46 @@ def _ets(bold_ts: np.ndarray) -> np.ndarray:
     u, v = np.tril_indices(n_roi, k=-1)
     # Compute the ETS
     return timeseries[:, u] * timeseries[:, v]
+
+
+def _correlate_dataframes(
+    df1: pd.DataFrame,
+    df2: pd.DataFrame,
+    method: Union[str, Callable] = "pearson",
+) -> pd.DataFrame:
+    """Column-wise correlations between two dataframes.
+
+    Correlates each column of `df1` with each column of `df2`.
+    Output is a dataframe of shape (df2.shape[1], df1.shape[1]).
+    It is required that number of rows are matched.
+
+    Parameters
+    ----------
+    df1 : pd.DataFrame
+        The first dataframe.
+    df2 : pd.DataFrame
+        The second dataframe.
+    method : str or callable, optional
+        any method that can be passed to:
+        https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.corr.html
```

Comment:
Can you use a ref here?
I think something like this should work:
``:func:`pandas.DataFrame.corr` ``